integration-cli/docker_cli_restart_test.go

```diff
@@ -81,69 +83,69 @@ func (s *DockerSuite) TestRestartDisconnectedContainer(c *check.C) {
 	// Run a container on the default bridge network
 	out, _ := dockerCmd(c, "run", "-d", "--name", "c0", "busybox", "top")
 	cleanedContainerID := strings.TrimSpace(out)
-	c.Assert(waitRun(cleanedContainerID), checker.IsNil)
+	assert.NilError(c, waitRun(cleanedContainerID))
 
 	// Disconnect the container from the network
-	out, err := dockerCmd(c, "network", "disconnect", "bridge", "c0")
-	c.Assert(err, check.NotNil, check.Commentf("%s", out))
```

Comment:
Interesting; looks like this was incorrect on multiple levels;

- `err` is actually an `exitCode` (`int`), so checking for `nil` / `not-nil` doesn't really make sense
- I rewrote it to check for a non-zero exit code (as that's what the code above appeared to be checking for), but it actually should run _succesfully_. The test passed, because `0 != nil` 🤷‍♂️